config/src/test-support/java/org/hyperledger/besu/config/StubGenesisConfigOptions.java

```diff
@@ -35,6 +35,11 @@
   private Optional<BigInteger> chainId = Optional.empty();
   private OptionalInt contractSizeLimit = OptionalInt.empty();
   private OptionalInt stackSizeLimit = OptionalInt.empty();
+  private OptionalLong defuseDifficultyBombBlockNumber = OptionalLong.empty();
```

Comment:
Again, if these could be ordered by when they occur in the blockchain it would help conceptually.